splice_ck/src/main/java/com/splicemachine/ck/HBaseInspector.java

```diff
@@ -54,41 +57,66 @@ public HBaseInspector(final Configuration config) {
         this.config = config;
     }
 
-    public String scanRow(final String region, final String rowKey, final Utils.SQLType[] cols) throws Exception {
+    String limitString(int limit)
+    {
+        return "   .\n   .\n   .\n" + Utils.Colored.red(
+                "--- Result list was limited to " + limit + " entries, " + "result was cut off ---");
+    }
+
+    public String scanRow(final String region, final String rowKey, final Utils.SQLType[] cols,
+                          int limit, int versions, boolean hbase) throws Exception {
+
         StringBuilder result = new StringBuilder();
-        try(final ConnectionWrapper connectionWrapper = new ConnectionWrapper();
-            final ResultScanner rs = connectionWrapper.withConfiguration(config).connect().withRegion(region).scanSingleRowAllVersions(rowKey)) {
-            TableRowPrinter rowVisitor = new TableRowPrinter(cols == null ? null : new UserDefinedDataDecoder(cols, 4));
+        UserDataDecoder decoder = cols == null ? null : new UserDefinedDataDecoder(cols, 4);
+
+        // scan for one more than the limit so now if we truncated
```

Comment:
Needs rephrasing maybe?

Reply:
done. sorry i meant "so that we KNOW if we truncated" not "now" :-P